### PR TITLE
trim down index

### PR DESCRIPTION
**Pull request by SerenaCaplins**

### CODE REVIEW

File: docs/source/welcome/index.md [hunk 1]
```diff
@@ -1,78 +1,40 @@
...
+:::{grid-item-card} {octicon}`info;1.5em;sd-mr-1` Getting Help
+:link: gettinghelp
+:link-type: doc
+
+We offer training resources and direct assitance as you access the computing cluster at NU.
++++
+[Learn more »](welcome/gettinghelp)
 :::
-:::{grid-item-card} {ref}`case-studies-and-testimonials`
+:::{grid-item-card} {octicon}`server;1.5em;sd-mr-1` Introduction to the cluster
```
Comment (joseph-cooke) [inline docs/source/welcome/index.md:24]: Should 'cluster' be 'Cluster'? Not sure on the case of items.

File: docs/source/welcome/services.md [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
+We encourage you to schedule a consultation with one of our staff members to receive personal, one-on-one assistance for your research computing, data storage, or classroom needs. Consultations are available to any Northeastern student, faculty, or staff member. We can help you get started on the cluster, optimize your code, benchmark, install and use software packages, detail data storage options, and more.
 
-We offer consultations by appointment most weekdays during regular business hours (9 a.m. to 5 p.m.). Just so you know, we follow the Northeastern University holiday schedule, so no consultations are available on holidays or during breaks. All consultations are conducted online through the Teams app.
+We offer consultations by appointment most weekdays during regular business hours (9 a.m. to 5 p.m.). We follow the Northeastern University holiday schedule, so no consultations are available on holidays or during breaks. All consultations are conducted online through the Teams app.
```
Comment (joseph-cooke) [inline docs/source/welcome/services.md:39]: Should it be application? Not sure if abbreviations should be used in documentation.

**Review by joseph-cooke — APPROVED** [review]
Looks really good!! I just left a few comments. Thank you!!

**Comment by gahecur** [comment]
The page looks good! I just reviewed it. I have thought about using abbreviations in the documents and I agree maybe we should avoid them.

**Review by gahecur — APPROVED** [review]

**Pull request merged**